superset/db_engine_specs/base.py

```diff
@@ -1304,6 +1304,30 @@ def get_column_spec(
             )
         return None
 
+    @classmethod
+    def get_cancel_query_id(cls, cursor: Any, query: Query) -> Optional[str]:
+        """
+        Select identifiers from the database engine that uniquely identifies the
+        queries to cancel. The identifier is typically a session id, process id
+        or similar.
+
+        :param cursor: Cursor instance in which the query will be executed
+        :param query: Query instance
+        :return: Query identifier
+        """
+        return None
+
+    @classmethod
+    def cancel_query(cls, cursor: Any, query: Query, cancel_query_id: str) -> None:
```

Comment:
Is it possible to return a boolean indicating if the query was canceled?

Reply:
addressed by 517c7928dfee2412b96836957a2efd2b5953c1e5